editor.planx.uk/src/@planx/components/shared/Schema/InputFields/shared.tsx

```diff
@@ -0,0 +1,14 @@
+import Typography from "@mui/material/Typography";
+import React from "react";
+
+interface FieldInputDescriptionProps {
+  description: string;
+}
+
+export const FieldInputDescription: React.FC<FieldInputDescriptionProps> = ({
+  description,
+}) => (
+  <Typography variant="body2" mb={1.5}>
+    {description}
+  </Typography>
+);
```

Comment:
**Two notes about this -** 
- When schema generation moves to the editor in the future, `description` will likely be rich text and we'll want to update this, but we don't need to account for this just yet 
- I'm also not 100% sure what a11y markup we need here - this is text that sits between the input label & input which are properly associated to each other, but the description itself is not currently associated to either of them
  - Maybe the input needs `aria-describedby` referencing a unique `id` set on this description?? 

I think the future probably looks more like this, but unless anyone knows for certain now let's revisit when we do a broader accessibility sweep of these new components soon!
```tsx
<Typography>
   <ReactMarkdownOrHtml
      source={description}
      openLinksOnNewTab
      // TODO a11y props? eg `id={DESCRIPTION_TEXT}` like CardHeader but unique because many fields on same page ??
    />
</Typography>
```

Reply:
For the note on `aria-describedby`, just for my knowledge. You use that to say this thing is described by element with `id = "..." ` so for this the description will have an `id` and the input would have the `aria-describedby`?

Reply:
Yes - I often reference MDN docs for clarification about ARIA properties, there's a lot of them & the differences are subtle! https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby